pages.ko/common/bash.md

```diff
@@ -1,32 +1,33 @@
 # bash
 
-> Bourne-Again SHell. `sh`-호환 명령 행 인터프리터.
-> 더 많은 정보: <https://gnu.org/software/bash/>.
+> Bourne-Again SHell, an `sh`- 호환 명령 행 인터프리터.
+> 참조 : `zsh`, `histexpand` (history expansion).
+> 더 많은 정보는 여기서 찾아볼 수 있습니다: <https://gnu.org/software/bash/>.
 
-- 대화식 쉘 시작:
+- 대화형 쉘 시작하기:
 
 `bash`
 
-- 명령 실행:
+- 설정 파일 로딩 없이 대화형 쉘 시작하기:
 
-`bash -c "{{command}}"`
+`bash --norc`
 
-- 파일에서 명령 실행:
+- 특정 명령어([c]ommands) 실행하기:
 
-`bash {{file.sh}}`
+`bash -c "{{echo 'bash is executed'}}"`
 
-- 파일에서 명령 실행하고, 터미널에서 실행 된 모든 명령 기록:
+- `path/to/script.sh` 스크립트 실행하기:
 
-`bash -x {{file.sh}}`
+`bash {{path/to/script.sh}}`
 
-- 파일에서 명령 실행하고, 첫 번째 에러에서 중지:
+- 각 명령어 실행 전 명령어 인쇄하며 `path/to/script.sh` 스크립트 실행하기:
```

Comment:
```suggestion
- 각 명령어 실행 전 명령어 인쇄하며 스크립트 실행하기:
```

Reply:
I thought you wanted to describe the specific file path https://github.com/tldr-pages/tldr/pull/8827#discussion_r990726295

Reply:
> @kbdharun I thought you wanted to describe the specific file path [#8827 (comment)](https://github.com/tldr-pages/tldr/pull/8827#discussion_r990726295)

In that comment, we were telling you to update the paths in command arguments inside `{{}}` not the command description.

Reply:
@kbdharun 
oh. misunderstood. you mean like below, right?

```
`bash -e {{path in Korean/to in Korean/script.sh}}` 
```

anyway, let me rollback the mentioning  ``{{path/to/script.sh}}` in the command description!

Reply:
Refer to #8820 if you need a reference for paths. You need to translate command arguments in line 21 and 25 to Korean.